plugins/woocommerce/tests/e2e-pw/tests/shopper/checkout.spec.js

```diff
@@ -140,8 +149,10 @@ test.describe( 'Checkout page', () => {
 		await expect( page.locator( 'strong.product-quantity' ) ).toContainText(
 			'2'
 		);
-		await expect( page.locator( 'td.product-total' ) ).toContainText(
-			twoProductPrice
+		let totalPrice = await page.getByRole( 'row', { name: 'Total' } ).last().locator( 'td' ).textContent();
+		totalPrice = Number( totalPrice.slice( 1 ).replace( /[^\d.-]/g, '' ) );
```

Comment:
Why do we do a `slice( 1 )` here? Shouldn't we just retrieve this value the same way we do in other places in this file?

Also, there is a lot of code duplication here, is there anyway we could reuse those repeated lines?

Reply:
Ah I missed one!

It's a 2 part question.  We don't need the `.slice( 1 )` because it's handled by the `.replace()` regex already.  So I'll remove it.  The reason this is being handled differently than in other places is because this locator is returning a string with the total price.  In the simplest cases, this would return `$9.99` (which we can't convert to a number).  In the worst case, it's returning something like `$9.99 (excluding $1.25 VAT)` or something like that. 

So we need to strip those strings down to just the numbers that can be converted to a float.  Playwright can only do a `.greaterThanOrEqual()` type of assertion on a numeric value.  Hence why we need to clean it up and convert it first.